admin-dev/themes/new-theme/js/nav_bar.js

```diff
@@ -22,158 +22,187 @@
+              .find('a > i.material-icons.sub-tabs-arrow').text('keyboard_arrow_up');
+            $('body').off('click.mobile');
+          }
+
+          $.post({
+            url: $(this).data('toggle-url'),
+            cache: false,
+            data: {
+              shouldCollapse: Number($('body').hasClass('page-sidebar-closed')),
+            },
+          });
         }
+      );
 
-        $.post({
-          url: $(this).data('toggle-url'),
-          cache: false,
-          data: {
-            shouldCollapse: Number($('body').hasClass('page-sidebar-closed'))
-          },
-        });
-      });
       addMobileBodyClickListener();
       const MAX_MOBILE_WIDTH = 1023;
 
       if ($(window).width() <= MAX_MOBILE_WIDTH) {
-          this.mobileNav(MAX_MOBILE_WIDTH);
+        this.mobileNav(MAX_MOBILE_WIDTH);
       }
 
       $(window).on('resize', () => {
-          if ($('body').hasClass('mobile') && $(window).width() > MAX_MOBILE_WIDTH) {
-              this.unbuildMobileMenu();
-          } else if (!$('body').hasClass('mobile') && $(window).width() <= MAX_MOBILE_WIDTH) {
-              this.mobileNav(MAX_MOBILE_WIDTH);
-          }
+        if ($('body').hasClass('mobile') && $(window).width() > MAX_MOBILE_WIDTH) {
+          this.unbuildMobileMenu();
+        } else if (!$('body').hasClass('mobile') && $(window).width() <= MAX_MOBILE_WIDTH) {
+          this.mobileNav(MAX_MOBILE_WIDTH);
+        }
       });
 
       function addMobileBodyClickListener() {
-          if (!$('body').is('.page-sidebar-closed:not(.mobile)')) {
-              return;
+        if (!$('body').is('.page-sidebar-closed:not(.mobile)')) {
+          return;
+        }
+        // To close submenu on mobile devices
+        $('body').on(
+          'click.mobile',
+          () => {
+            if ($('ul.main-menu li.ul-open').length > 0) {
+              $('.nav-bar li.link-levelone.has_submenu.ul-open').removeClass('ul-open open -hover');
+              $('.nav-bar li.link-levelone.has_submenu.ul-open ul.submenu').removeAttr('style');
+            }
           }
-          // To close submenu on mobile devices
-          $('body').on('click.mobile', function() {
-              if ($('ul.main-menu li.ul-open').length > 0) {
-                  $('.nav-bar li.link-levelone.has_submenu.ul-open').removeClass('ul-open open -hover');
-                  $('.nav-bar li.link-levelone.has_submenu.ul-open ul.submenu').removeAttr('style');
-              }
-          });
+        );
       }
     });
   }
 
   mobileNav() {
-    let $logout = $('#header_logout').addClass('link').removeClass('m-t-1').prop('outerHTML');
-    let $employee = $('.employee_avatar').prop('outerHTML');
-    let profileLink = $('.profile-link').attr('href');
+    const $logout = $('#header_logout').addClass('link').removeClass('m-t-1').prop('outerHTML');
+    const $employee = $('.employee_avatar').prop('outerHTML');
+    const profileLink = $('.profile-link').attr('href');
```

Comment:
Question: what is the meaning of adding a '$' at the beginning of a variable, does it mean that it mean that **.prop('outerHTML')** returns a jquery object?

Reply:
I don't know :D as I know it's not returning an HTML object.

Reply:
This is what happened when there are no rules :trollface:

Reply:
haha yes indeed, it's not meaningless if the goal is to give a cue about the type of the variable (a jquery object), but here I'm not sure. No big deal since those variables were already there and used...